modules/tensor_mechanics/include/materials/ComputePlasticTrussResultants.h

```diff
@@ -0,0 +1,71 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ComputeTrussResultants.h"
+
+class ComputePlasticTrussResultants : public ComputeTrussResultants
+{
+public:
+  static InputParameters validParams();
+
+  ComputePlasticTrussResultants(const InputParameters & parameters);
+
+protected:
+  virtual void computeQpProperties();
+  virtual void initQpStatefulProperties();
+
+  virtual Real computeHardeningValue(Real scalar);
+  virtual Real computeHardeningDerivative(Real scalar);
+
+  //  yield stress and hardening property input
```

Comment:
should go further down and have three `/`

Reply:
I have corrected it now.